Otec/LoginViewController.swift

```diff
@@ -23,6 +23,16 @@ class LoginViewController: UIViewController {
         }
     }
     
+    private func checkIfHasSuscriptions() -> Bool {
```

Comment:
I'd improve this function's name by removing the word `check` (which, in most of cases, should be unnecessary) and add the *what* is checking against. For example: `newspapersHaveNewSuscriptions()`.
 `thereAreNewSubscriptions()` would also work, I guess.